embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/LogsApi.kt

```diff
@@ -121,8 +125,10 @@ void logException(@NonNull Throwable throwable,
      * @param stacktraceElements the stacktrace to remotely log
      * @param severity           the severity level of the log message
      */
-    void logCustomStacktrace(@NonNull StackTraceElement[] stacktraceElements,
-                             @NonNull Severity severity);
+    fun logCustomStacktrace(
+        stacktraceElements: Array<StackTraceElement?>,
```

Comment:
Do we actually allow nullable elements in the array?

Reply:
Good catch... updated.